homework/homework.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"gocourse05/core/camera"
+	"math/rand/v2"
+)
+
+type ID interface {
+	GetId() int
+}
+
+type Metadata interface {
+	GetMetaData() string
+}
+
+func main() {
+	camerasList := buildCamerasList()
```

Comment:
"list" зайве:
```suggestion
	cameras := buildCameras()
```

https://google.github.io/styleguide/go/decisions#variable-names

> Omit [types and type-like words](https://google.github.io/styleguide/go/decisions#repetitive-with-type) from most variable names

Reply:
цікаво, скоротив